# goal definition
geometry_msgs/Point position
---
# result definition
bool success_pos
bool success_yaw
---
# feedback
geometry_msgs/Point position
string state